Don't show track view if there is only one track

Arnei opened the pull request:
If an event has only a single track, you cannot deselect that track, so the track view is rather pointless in that case. Therefore, we might as well not show it.

Fixes #1205.

Arnei committed:
Don't show track view if there is only one track

If an event has only a single track, you cannot deselect that track,
so the track view is rather pointless in that case. Therefore, we might
as well not show it.

LukasKalbertodt reviewed (approved):
Code looks good and seems to work (see https://test.editor.opencast.org/1209/2023-11-07_09-24-17/?mediaPackageId=ID-nasa-earth-4k)

lkiesow commented:
It might still be confusing to users when part of the tooling just silently disappears.
Maybe just adding an info box to the track selection would actually be better?
There you could put that this tool is disabled and why.

Arnei committed:
Merge remote-tracking branch 'upstream/main' into dont-show-track-selection

KatrinIhler reviewed (changes requested):
Here only the info box thingy is still outstanding. Requesting changes to accurately represent the state.

Arnei committed:
Show info box instead of unactionable track selection

Reverts previous change of not showing the
track selection at all, and instead shows
an info box in the track selection view.

Arnei commented:
Basic info box. Suggestions for phrasing are welcome :)

![Bildschirmfoto vom 2024-05-13 15-45-26](https://github.com/opencast/opencast-editor/assets/14070005/b560e4c7-addd-43c6-b67d-876e8f418ca7)

Arnei committed:
Merge branch 'main' into dont-show-track-selection

Arnei committed:
Merge branch 'main' into dont-show-track-selection

dennis531 commented on src/i18n/locales/en-US.json:
```diff
@@ -240,6 +240,14 @@
+      "deleteTrackTooltip": "Do not encode and publish this track.",
+      "restoreTrackTooltip": "Encode and publish this track.",
+      "cannotDeleteTrackTooltip": "Cannot remove this track from publication.",
+      "noSelectionPossible": "Track selection is disabled for this video, because there are not enough tracks to deselect any.",
```
I would suggest providing a more precise description about the number of tracks required:

```suggestion
      "noSelectionPossible": "Track selection is disabled for this video, because at least two tracks are required to deselect any.",
```

Arnei committed:
Make text more precise

Co-authored-by: Dennis Benz <dennis.benz@uni-osnabrueck.de>

Arnei committed:
Merge branch 'main' into dont-show-track-selection

Arnei committed:
Fix eslint
